stubs/livewire/resources/views/profile/remove-connected-accounts-form.blade.php

```diff
@@ -0,0 +1,101 @@
+            @endif
+        </h3>
+
+        <div class="mt-3 max-w-xl text-sm text-gray-600">
+            @if (count($this->accounts) == 0)
+                {{ __('When you have one or more connected accounts, they will appear below. You may disconnect any of your connected accounts at any time. If you feel any of your connected accounts have been compromised, you should disconnect them immediately and change your password.') }}
+            @else
+                {{ __('You may disconnect any of your connected accounts below at any time. If you feel any of your connected accounts have been compromised, you should disconnect them immediately and change your password.') }}
+            @endif
+        </div>
+
+        @if (count($this->accounts) > 0)
+            <div class="mt-5 space-y-6">
+                @foreach($this->accounts as $account)
+                    <div class="p-3 flex items-center justify-between">
+                        <div class="flex items-center">
+
+                            @switch($account->provider_name)
+                                @case('facebook')
+                                    <x-jet-facebook-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @case('google')
+                                    <x-jet-google-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @case('twitter')
+                                    <x-jet-twitter-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @case('linkedin')
+                                    <x-jet-linkedin-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @case('github')
+                                    <x-jet-github-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @case('gitlab')
+                                    <x-jet-gitlab-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @case('bitbucket')
+                                    <x-jet-bitbucket-icon class="h-6 w-6 mr-2" />
+                                    @break
+                                @default
+                            @endswitch
+
+                            <div>
+                                <div class="text-sm font-semibold text-gray-600">
+                                    {{ ucfirst($account->provider_name) }}
+                                </div>
+
+                                <div class="text-xs text-gray-500">
+                                    Added on {{ $account->created_at }}
+                                </div>
+                            </div>
+                        </div>
+
+                        <!-- Remove Action -->
+                        @if(count($this->accounts) > 1)
```

Comment:
For consistency can you go through your PR and make sure that all `@if (` directives have a space between the `if` and `(`?